### ISSUE TITLE
torch.onnx.export fails on Linear when bias=False, torch.onnx.symbolic.matmul does not exist

### ISSUE THREAD

**Issue by Erotemic** [pytorch/pytorch#4408]
Using `torch.onnx.export` on a model that contains a linear layer with no bias, I receive an error: 

```python
~/venv3/lib/python3.5/site-packages/torch/onnx/__init__.py:312: UserWarning: ONNX export failed on matmul because torch.onnx.symbolic.matmul does not exist
  File "~/venv3/lib/python3.5/site-packages/torch/onnx/__init__.py", line 151, in _export
    proto = trace.export([], _onnx_opset_version)
RuntimeError: ONNX export failed: Couldn't export operator matmul

Graph we tried to export:
graph(%0 : Float(1, 128)
      %1 : Float(2, 128)) {
  %2 : Float(128!, 2!) = Transpose[perm=[1, 0]](%1), scope: Linear
  %3 : Float(1, 2) = matmul(%0, %2), scope: Linear
  return (%3);
}
```

I've constructed a minimal working example that reproduces this
```python
def mwe_bias():
    linear = torch.nn.Linear(128, 2, bias=1)
    args = tuple([torch.autograd.Variable(torch.randn(1, 128), requires_grad=True)])
    torch.onnx.export(linear, args, 'foo.onnx', export_params=False)


def mwe_nobias():
    linear = torch.nn.Linear(128, 2, bias=0)
    args = tuple([torch.autograd.Variable(torch.randn(1, 128), requires_grad=True)])
    torch.onnx.export(linear, args, 'foo.onnx', export_params=False)

# does not cause an error
mwe_bias()
# Causes an error
mwe_nobias()
```

Environment Info:
```python
sys.platform = linux
sys.version = 3.5.2 (default, Nov 17 2016, 17:05:23) \n[GCC 5.4.0 20160609]
torch.version.__version__ = '0.4.0a0+af3bffb'
torch.version.cuda = 8.0.61
```

**Comment by Erotemic** [comment]
I notice in `symbolic.py` there is a function: 

```
def bmm(g, self, other):
    return g.op("MatMul", self, other)
```

is it sufficient to simply add another function: 

```
def matmul(g, self, other):
    return g.op("MatMul", self, other)
```

If I hack this into my installed pytorch, then the model "seems" to work correctly. I can save it, reload and then it checks correctly. 

Is this the correct fix? I'm not really sure what `bmm` stnads for, nor is it obvious from the docs what sort of object `g` is (although I assume it might stand for graph?).

**Comment by apaszke** [comment]
`bmm` is batched matrix multiply, `g` is a graph. I'm not 100% sure that this is enough, but it looks good. Can you please send a PR, and we'll have someone review that? Thank you!

**Comment by soumith** [comment]
fixed via #4452
